RadarSDK/RadarSettings.h

```diff
@@ -65,7 +65,6 @@ NS_ASSUME_NONNULL_BEGIN
 + (void)updateLastAppOpenTime;
 + (NSDate *)lastAppOpenTime;
 + (BOOL)useRadarModifiedBeacon;
-+ (BOOL)useLocationMetadata;
```

Comment:
If we totally rip this out in this PR, we'll need to proactively let people using `useLocationMetadata` know.